src/uct/ib/rc/accel/rc_mlx5_devx.c

```diff
@@ -471,9 +473,14 @@ uct_rc_mlx5_iface_common_devx_connect_qp(uct_rc_mlx5_iface_common_t *iface,
         }
     }
 
+    /* Do not take max_rd_atomic from iface config, because this function
```

Comment:
maybe fix iface->super.config.max_rd_atomic so it will be correct also for DC?

Reply:
it is already correct for DC, that's why it fails here. Here RC QP is created - we always create command QP as RC QP even with DC transport. The problem is that we take DC value for creating RC QP

Reply:
Can we pass a parameter to this function so that for command QP it will be 0 or 1 for example?
because we don't want to ignore UCX_RC_MAX_RD_ATOMIC config

Reply:
ic your point, will update